src/protections/WordList.ts

```diff
@@ -138,6 +138,11 @@ export class WordListProtection
     room: MatrixRoomID,
     event: RoomEvent
   ): Promise<ActionResult<void>> {
+    // If the sender is draupnir, ignore the message
+    if (event["sender"] === this.draupnir.clientUserID) {
+      log.info(`Ignoring message from self: ${event.event_id}`);
```

Comment:
whoop, I got too excited, but it's cool and optimistic merging is good. However, these logs would have been maybe ok if they were `debug` rather than `info` (draupnir could get pretty noisy with these protections since we send a lot of events). Personally though I wouldn't have logged at all but there is merit to logging as it would help someone debugging something in future maybe.

Reply:
I can fix this for you if you like?

Reply:
It'd be faster for you to fix it. I noticed I'd left it as INFO almost immediately after you merged lmao

Reply:
okie, i can do that tomorrow ^^